docs/3.1. Discovery - Registered Operation.md

```diff
@@ -44,13 +44,13 @@ Values 0 to 99 correspond to an active NMOS Registration API (zero being the hig
 
 If the chosen Registration API does not respond correctly at any time, another Registration API should be selected from the discovered list. Should no further Registration APIs be available or TTLs on advertised services expired, a re-query may be performed.
 
-If no Registration APIs are advertised on a network, the Node should assume peer to peer operation unless configured otherwise. The required TXT record advertisements for this mode are described in the [Peer to Peer Operation](3.2.%20Discovery%20-%20Peer%20to%20Peer%20Operation.md) documentation.
+If no Registration APIs are advertised on a network, the Node should assume peer-to-peer operation unless configured otherwise. The required TXT record advertisements for this mode are described in the [peer-to-peer Operation](3.2.%20Discovery%20-%20Peer%20to%20Peer%20Operation.md) documentation.
```

Comment:
(I'm sorry I started this!) Peer-to-Peer capitalized here.

Reply:
Clearly I was a little careless with the find and replace.